nginx/conf.d/production.conf

```diff
@@ -0,0 +1,52 @@
+server {
+    listen 80;
+    listen 443 ssl;
+    server_name localhost;
+    root /var/www/html;
+
+    ssl_certificate /etc/nginx/ssl/certs/localhost.crt;
```

Comment:
@tyrossel This production conf is not well updated right ? Because `server_name` etc are `localhost`, but this will not be `localhost` on prod